files/ko/web/http/headers/referer/index.html

```diff
@@ -1,40 +1,43 @@
 ---
 title: Referer
 slug: Web/HTTP/Headers/Referer
+tags:
+  - HTTP
+  - Reference
+  - header
+  - referer
+  - referrer
 translation_of: Web/HTTP/Headers/Referer
+browser-compat: http.headers.Referer
 ---
 <div>{{HTTPSidebar}}</div>
 
-<p><code><strong>Referer</strong></code> 요청 헤더는 현재 요청된 페이지의 링크 이전의 웹 페이지 주소를 포함합니다. <code>Referer</code> 헤더는 사람들이 어디로부터 와서 방문 중인지를 인식할 수 있도록 해주며 해당 데이터는 예를 들어, 분석, 로깅, 혹은 캐싱 최적화에 사용될 수도 있습니다.</p>
+<p><code><strong>Referer</strong></code> 요청 헤더는 현재 요청된 페이지의 링크 이전의 웹 페이지 주소를 포함합니다. <code>Referer</code> 헤더는 사람들이 어디로부터 와서 방문 중인지를 인식할 수 있도록 해주며 해당 데이터는 예를 들어, 분석, 로깅, 혹은 캐싱 최적화에 사용될 수도 있습니다.</p>
 
-<p>referer는 실제로 단어 "referrer"에서 철자를 빼먹은 것입니다. 자세한 내용은 {{interwiki("wikipedia", "HTTP_referer", "HTTP referer on Wikipedia")}}을 참고하세요.</p>
+<p><code>Referer</code> 헤더는 URL 프래그먼트 (예 : "#section") 또는 "username : password" 정보를 포함 할 수 없습니다. 잠재적으로 <em>원본</em>, <em>경로</em>, 및 <em>쿼리 문자열</em>을 포함 할 수 있습니다. 전송되는 내용은 요청에 대한 referer 정책에 따라 다릅니다. 정보 및 예제는 Referrer-Policy를 참조하십시오. <a href="/en-US/docs/Web/HTTP/Headers/Referrer-Policy#directives">정보</a> 와 <a href="/en-US/docs/Web/HTTP/Headers/Referrer-Policy#examples">예시</a>는 이곳 {{HTTPHeader("Referrer-Policy")}}을 참고하세요.
```

Comment:
origin의 경우 헤더의 종류 중 하나인 점과, mdn origin 페이지에서 '원본'으로 표기하지 않는 것으로 파악되어 원본 대신 origin이라고 그대로 표기하면 어떨지 제안 드려봅니다 :)
그 외 referrer policy 오타 수정하였고, potentially 라는 단어가 잠재적이라는 뜻보단 possibly에 가깝게 느껴져 어감을 수정하면 어떨까 싶어 이 또한 제안 드립니다 :)

```suggestion
<p><code>Referer</code> 헤더는 URL 프래그먼트 (예 : "#section") 또는 "username : password" 정보를 포함 할 수 없습니다. <em>origin</em>, <em>경로</em>, 및 <em>쿼리 문자열</em>을 포함 할 수는 있습니다. 전송되는 내용은 요청에 대한 referrer 정책에 따라 다릅니다. 정보 및 예제는 Referrer-Policy를 참조하십시오. <a href="/en-US/docs/Web/HTTP/Headers/Referrer-Policy#directives">정보</a> 와 <a href="/en-US/docs/Web/HTTP/Headers/Referrer-Policy#examples">예시</a>는 이곳 {{HTTPHeader("Referrer-Policy")}}을 참고하세요.
```